common/gsutil.py

```diff
@@ -19,43 +19,46 @@
 logger = logs.Logger('gsutil')
 
 
-def gsutil_command(arguments, *args, parallel=False, **kwargs):
+def gsutil_command(arguments, expect_zero=True, parallel=False):
     """Executes a gsutil command with |arguments| and returns the result."""
     command = ['gsutil']
     if parallel:
         command.append('-m')
-    return new_process.execute(command + arguments, *args, **kwargs)
+    return new_process.execute(command + arguments, expect_zero=expect_zero)
 
 
-def cp(*cp_arguments, **kwargs):  # pylint: disable=invalid-name
+def cp(source, destination, parallel, recursive=False):  # pylint: disable=invalid-name
```

Comment:
What about putting a default value for `parallel=False` as `rm` and `rsync`?

Reply:
+1